GeneratedRegexPatterns/MediumConfidenceSecurityModels.json

```diff
@@ -20,7 +20,7 @@
     "DetectionMetadata": "HighEntropy, MediumConfidence"
   },
   {
-    "Pattern": "(ftps?|https?):\\/\\/(?<refine>[^:@\\/]+:[^:@?\\/]+)@",
+    "Pattern": "($|\\b)(ftps?|https?):\\/\\/(?<refine>[^:@\\/]+:[^:@?\\/]+)@",
```

Comment:
How are we testing this change to the regex?

Reply:
Did you see the test updates? Let me ensure I pushed them.

Reply:
Yes, the tests are further in this file. We have a pattern in this code of generating test data in the rule itself.